downstairs/src/main.rs

```diff
@@ -239,6 +255,51 @@ async fn main() -> Result<()> {
     let log = build_logger();
 
     match args {
+        Args::Clone {
+            data,
+            source,
+            trace_endpoint,
+        } => {
+            /*
```

Comment:
Is this necessary, since we compile with `panic = abort`?

Reply:
I dragged it from the other paths in main, but, yeah, given the top level, I don't think
we need this anywhere.

I'll file another issue to remove it from the 11 other places, no need to dirty up this PR with
those changes.